### DISCUSSION TITLE
Edit multiple books metadata (tags, series, etc) at the same time.

### DISCUSSION THREAD

**Discussion by kamalsoor** [advplyr/audiobookshelf#1950] [Ideas]
Edit multiple books metadata (tags, series, etc) at the same time.      

I’m moving my library over from Plex  to AudioBooksShelf (ABS), since it’s better at handling series etc.   In Plex, I’m able to quickly apply changes to multiple books at the SAME time.  Within ABS it’s very slow process since I have to update each book and hit save for each one. The way my files are organized mean that ABS is picking up Series automatically or the Authors name is coming across with different spelling, (ie John le Carré, John le Carre, John Le Carré)

Here is how Plex does it.   Can this be done with ABS?    One Window to Rule them All :) 

Select files in Plex:
<img width="793" alt="main_select" src="https://github.com/advplyr/audiobookshelf/assets/7356572/0dc04d8d-c0c2-4fb1-b7c4-762059919a0f">

Plex Edit files :
<img width="771" alt="plex edit" src="https://github.com/advplyr/audiobookshelf/assets/7356572/9d3716b4-3e35-4a4a-9d1f-bb66db37ad60">

**Comment by advplyr** [comment]
Abs already supports this I think. What is different from Abs?

**Reply by kamalsoor** [reply]
My Bad! Sorry, I didn’t notice the “Map” drop down.   As a new user my eye goes to all the book panels.  
Thanks for your quick reply.  

K
